### PR TITLE
Cleanup admin settings js files

### PR DESCRIPTION
**Pull request by CarlSchwan**
* Move admin.js to webpack, so that this uses the bundled jquery files
  instead of the deprecated window.$ Also fixing formatting

* Remove log.js that seems to be from a time when logreader was bundled
  with server and I couldn't find an usage of it anymore

### CODE REVIEW

**Review by artonge — APPROVED** [review]
Nice, how many warnings does this remove? :smile:

**Review by MichaIng — APPROVED** [review]

File: apps/settings/src/store/apps.js [hunk 1]
```diff
@@ -180,16 +182,16 @@ const actions = {
...
 									),
-									t('settings', 'App update'),
-									function() {
-										window.location.reload()
-									},
-									true
+									{
+										onClick: () => window.location.reload(),
+										close: false,
+
```
Comment (MichaIng) [inline apps/settings/src/store/apps.js:193]:
```suggestion
```

File: apps/settings/js/apps.js [hunk 2]
```diff
@@ -1,113 +0,0 @@
-/* global Handlebars */
```
Comment (artonge) [inline apps/settings/js/apps.js:1]: Does nobody else use `OC.Settings.Apps.rebuildNavigation()`?

File: apps/settings/css/settings.scss [hunk 3]
```diff
@@ -1755,3 +1755,18 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
 		}
 	}
 }
+
```
Comment (artonge) [inline apps/settings/css/settings.scss:1758]: Where does this come from?

Comment (CarlSchwan) [reply apps/settings/css/settings.scss:1758]: To have the blinking animation when adding a new app entry inside the navbar. This was previously dine with jQuery animations.

File: apps/settings/js/apps.js (hunk 2 above)
Comment (CarlSchwan) [reply apps/settings/js/apps.js:1]: Not in server and the multiple file related apps installed locally. And it's a private api (OC instead of OCP)

Comment (artonge) [reply apps/settings/js/apps.js:1]: Not sure if we support external_sites: https://github.com/nextcloud/external/blob/2cde1b31c07b09c91ffd5750cadc6e31c477ebb9/src/admin.js#L211

Comment (CarlSchwan) [reply apps/settings/js/apps.js:1]: Let me add a compatibility alias 😞

Comment (CarlSchwan) [reply apps/settings/js/apps.js:1]: done and tested with the external app

**Pull request merged**